lib/three_scale/sidekiq_batch_cleanup_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'progress_counter'
+
+require Rails.root.join('app/lib/three_scale/patterns/service')
```

Comment:
So @akostadinov I don't like this `require`...
I think ideally we shouldn't require `app/lib/` modules from `lib/`, I'd say it should be the other way around. Or maybe we should move the patterns from `app/lib` to `lib/` 

In general, I don't quite understand the structure of modules in this project. What's `lib`? what's `app/lib`? Why some modules are in `ThreeScale` namespace, and others are not... I don't know, this is all confusing...

Reply:
but yeah, perhaps can be moved out of `app`. But needs to be imported somewhere else too